src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,76 @@
+.container--color {
+   background-color:var(--bisque); 
+}
+.footer {
+    padding: 10% 8%;
+    background-color:var(--bisque);
+    @include breakpoint (lg) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 5% 5%;
+      }
+    @include breakpoint (xl) {
+        margin: 0 auto;
+        width: 100%;
+        max-width: 1440px;
+    }
+    &__logo {
+        margin-left: 12%;
+        margin-right: 12%;
+        @include breakpoint (lg) {
+            width: 18%;
+            margin-bottom: -78px;
+        }
+    }
+    &__nav {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-family: Coolvetica;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 18px;
+        line-height: 120%;
+        text-align: center;
+        letter-spacing: 0.02em;
+        @include breakpoint (lg) {
+            flex-direction: row;
+            width: 94%;
+            justify-content: space-between;
+            & li:nth-child(2)  {
+                padding-right: 16%;
+            }
+            & li:nth-child(3)  {
+                padding-left: 16%;
+            }
+        }
+        &--space {
+            margin-top: 48px;
+        }
+    }
+    &__line {
+        margin-top: 40px;
+        margin-bottom: 40px;
+        height: 0.5px;
+        border: none;
+        background-color:var(--grey);
+        @include breakpoint (lg) {
+            width: 100%;
+            }
+      }
+    &__social {
+        display: flex;
+        justify-content: center;
+    }
+    &__copyright {
+        font-size: 15px;
```

Comment:
Use rem units here. It would be `font-size: 1.5rem;`

Reply:
Also, according to the design the `font-family` needed here is `Coolvetica`. You can use the variable defined for that. It would be something like `font-family: $font-family-regular;`

Reply:
The color of this text is `#747474`. It is not in the `_variables.scss` file, so you need to add it there and use it here. Something like `color: $new-color;`

Reply:
The font-size is 15px according to figma designs.
![image](https://user-images.githubusercontent.com/68356379/130700631-629c6e11-6ecd-474f-8b67-8c41bb16f5bf.png)

Reply:
Done